chromadb/utils/embedding_functions/wordllama_embedding_function.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Dict, Literal, cast
+
+from chromadb.api.types import Documents, EmbeddingFunction, Embeddings
+
+if TYPE_CHECKING:
+    from wordllama.inference import WordLlamaInference
+
+
+class WordLlamaEmbeddingFunction(EmbeddingFunction[Documents]):
+    models: Dict[str, WordLlamaInference] = {}
+
+    def __init__(
+        self,
+        config: Literal["l2_supercat", "l3_supercat"] = "l2_supercat",
```

Comment:
Can we stick to standard naming conventions and use `model_name` instead?

Reply:
It's not really a model name, it's a config name which is shorthand for model+some parameters.

And it can be used to load configurations from TOML files, so it's better this way.

Reply:
Good point. Thanks for clarifying. Can you add this to the param description. Or provide a link in the description to point users where to find more info about it.

We lean heavy on DX so that's why the param name struck with me that for some users might cause a confusion unless explained how it works and where to find info.

Reply:
That's a tricky one.

Source code for reference: https://github.com/dleemiller/WordLlama/blob/main/wordllama/wordllama.py

WordLlama stores several predefined configs, but publicly mentions only those two, and i'm not sure if describing what it is here is a good idea.
And it also accepts `WordLlamaConfig`, but really with no docs or examples.

From what I see, other configs are used primarily for training, to produce these two fast and small models.

Maybe it's better to leave them just as "configs" with current description, because most intended use-cases use only these parameters anyway.

And, to be precise, `load(config: ...` accepts `str | WordLlamaConfig`, so we have a choice between describing it here as Literal with two expected choices, or as `str` and trying to explain the details user probably does not need (and i may be wrong, because I used only these two choices).

Maybe we should leave this description as it is. I only replaced word 'model' with 'config', to be consistent.

Reply:
>  If you have online docs, link them in the file header.

Added description from the repository to class docstring, and also added a link.

(Added as docstring, and not a file header, because other files mostly using docstrings)

Reply:
@tazarov please check this again.